std/conv.d

```diff
@@ -5653,17 +5665,19 @@ auto toChars(ubyte radix = 10, Char = char, LetterCase letterCase = LetterCase.l
             {
                 Result result = void;
                 result.buf = buf;
-                result.lwr = cast(ubyte)(this.lwr + lwr);
-                result.upr = cast(ubyte)(this.lwr + upr);
+                result.lwr = cast(uint)(this.lwr + lwr);
+                result.upr = cast(uint)(this.lwr + upr);
                 return result;
             }
 
           private:
+            uint lwr = void, upr = void;
             char[(UT.sizeof == 4) ? 10 + isSigned!T : 20] buf;
```

Comment:
guess it doesn't matter because your `Result` starts with void anyway, but then the `= void` in lwr and upr are superfluous